packages/suite/src/reducers/suite/suiteReducer.ts

```diff
@@ -87,7 +87,7 @@ export interface SuiteSettings {
     isDeviceAuthenticityCheckDisabled: boolean;
     addressDisplayType: AddressDisplayOptions;
     defaultWalletLoading: WalletType;
-    experimental?: ExperimentalFeature[];
+    experimental?: ExperimentalFeatureConfig['id'][];
```

Comment:
There ain't no such thing as `ExperimentalFeatureConfig['id']`. Revert back to `ExperimentalFeature`?

Reply:
fixed